java/openpkw-services/src/main/java/org/openpkw/mapping/PeripheralCommitteeFieldSetMapper.java

```diff
@@ -0,0 +1,41 @@
+package org.openpkw.mapping;
+
+import org.openpkw.model.entity.PeripheralCommittee;
+import org.springframework.batch.item.file.mapping.FieldSetMapper;
+import org.springframework.batch.item.file.transform.FieldSet;
+import org.springframework.validation.BindException;
+
+import java.util.Arrays;
+
+/**
+ * Created by wojciech.szostak on 14.08.2015.
+ */
+public class PeripheralCommitteeFieldSetMapper implements FieldSetMapper<PeripheralCommittee> {
+    @Override
+    public PeripheralCommittee mapFieldSet(FieldSet fieldSet) throws BindException {
+        PeripheralCommittee peripheralCommittee = new PeripheralCommittee();
+
+        try {
+            peripheralCommittee.setPkwId(fieldSet.readString(0));
+            peripheralCommittee.setTerritoryCode(fieldSet.readLong(4));
+            peripheralCommittee.setType(fieldSet.readString(5));
+            peripheralCommittee.setName(fieldSet.readString(15));
+            try {
+                peripheralCommittee.setAllowedToVote(fieldSet.readLong(16));
```

Comment:
Tu chyba lepszym pomysłem było by chyba zrobienie metody sprawdzającej z odpowiednią nazwą np. isAllowedToVote() i puszczenie ifa.  Łączenie logiki w try, catch jest słabe, powinno to służyć wyłączenie do obsługi wyjątków. Do zastanowienia też czy warunek powinien być w tym try- catch.